vpr/src/route/route_common.cpp

```diff
@@ -93,11 +93,10 @@ static vtr::vector<ClusterNetId, std::vector<int>> load_net_rr_terminals(const t
 static vtr::vector<ClusterBlockId, std::vector<int>> load_rr_clb_sources(const t_rr_node_indices& L_rr_node_indices);
 
 static t_clb_opins_used alloc_and_load_clb_opins_used_locally();
-static void adjust_one_rr_occ_and_apcost(int inode, int add_or_sub, float pres_fac, float acc_fac);
+static void adjust_one_rr_occ_and_apcost(int inode, int add_or_sub, float acc_fac);
```

Comment:
Should rename to adjust_one_rr_occ_and_acc_cost (no longer a present congestion cost update, so apcost doesn't make sense anymore).